### DISCUSSION TITLE
New green software training for product owners, product managers, business analysts

### DISCUSSION THREAD

**Discussion by holanita** [Green-Software-Foundation/learn#165] [General]
@VirginieCorraze (pending invitation) suggested additional training during the [Community WG call today](https://github.com/Green-Software-Foundation/community-wg/issues/48). 
Discussion to be continued in this repo.
@greenhsu123 @tmcclell

**Comment by VirginieCorraze** [comment]
@holanita @jawache following our discussion at the community workgroup yesterday:
It would be very valuable to have an e-certification covering the early phases of the solutions design, to take the right decisions and influence from the very beginning:
•	GREEN BY DESIGN: eco-design practices to adopt at the early stages of the product design (scoping phase, high-level requirements, mock-up)  

•	Targeted audience: Product Owner, Product manager, Business analyst

•	How to manage the design and delivery of IT solutions by minimizing environmental impact 
•	The right questions to consider, how to arbitrate
•	How to determine the arguments to embark the internal stakeholders and customers on a responsible aproach.
•	How to make trade-offs according to your priorities
•	Define the key performance indicators and measurement to achieve your goals
•	Role and responsibly of the Product Owner

**Reply by jawache** [reply]
Adding in @PindyBhullar also because I've heard her talk along similar lines and also @benhamidene commented below that this is something he'd be interested in supporting.

I don't believe what's being proposed is an addition to the existing training but completely different training, it's a more targeted audience, whereas our original training is for everyone.

I propose we carry on the discussion here and if a more firm proposal is reached we pitch it to the oversight committee and relevant working groups.

- Question: Does the audience for this include UX Design or is that a separate discipline to this?
- Question: Is there existing training in this realm? (I remember there was some training on the Salesforce training portal called eco-design but that was more for UX design I believe).
- Question: Is eco-design is an overloaded term here? I think it's just another word for "green software" in this context, perhaps we're talking about "green software principles for <a name for this audience>"

**Reply by benhamidene** [reply]
To the question "_Does the audience for this include UX Design or is that a separate discipline to this?_"

I think UX Design is part of it but only until a certain depth . Product Owners need to know the impact of UX Design on the environmental foot print of their products  , but they don't need to work it out in detail

**Reply by benhamidene** [reply]
To the Question "_Is eco-design is an overloaded term here? I think it's just another word for "green software" in this context, perhaps we're talking about "green software principles for "_"

Well one of the goals of the training is to provide answers for "_How to manage the design and delivery of IT solutions by minimizing environmental impact_"
I understand that we are not only talking about the design of the product itself but also about the delivery process that needs to be managed and considered by the product owner. This opens new windows to more challenges I suppose.
Or am I extending the scope with this ?

**Reply by jawache** [reply]
Thanks @benhamidene I think first step for this is to identify the audience for this (rather than the content). Given the answers above and @VirginieCorraze's original statement I think her identification of the audience as "Product Managers, Project Managers and Business Analysts" is correct. I was asking about UX Designers since the topic was "Eco Design" and wondering if the intention was to speak to UX Designers but again based on the answers above I don't believe our intention is to speak to UX Designers with this content.

I would suggest we perhaps treat this as a certification rather than creating content.

The nature of the Foundation means it's very hard to create training like we did for learn.greensoftware.org, takes many many months of effort and everything needs to go through multiple levels of consensus review, it's pretty tiring.

@camcash17 was exploring the idea of just creating the exam for a topic, we create and administer the exam and point people to existing content for the training. If we realize that some content for some exam questions doesn't exist we can create the missing content, or perhaps even outsource or somehow encourage the community to create the content to fill in the gaps. I much prefer that solution, it's very challenging to get multiple people to agree on content I suspect it's going to be much easier to get multiple people to agree to some exam questions. 

So I propose we talk about this project in the context of and exam "Certified Green Software Project/Product Manager" (or some term which accurately defines the role).

If we discover during the creation of the exam that much of the required reading/content doesn't exist we can explore options for filling in the gaps and making sure the content exists. But our goal would be just to administer the exam.

Thoughts?

**Reply by PindyBhullar** [reply]
We do need to differentiate between the planning of technology solutions, as the key decisions are made at the outset of any project. When the software is being developed it becomes a finer granular coding styles and testing frameworks. 
The key point made be @VirginieCorraze is how to make the trade off between priorities, and to add to this is the costs associated with them too. The institutional logics (priorities) that are are in play in organisations is the challenge. 

@jawache comments re: training is also worth exploring. The training provided today, is geared towards the carbon SDK and the SCI (rightly so). It helps standardize the language around green IT and carbon measurement. 
But this training is the foundation of the baseline of collecting energy metrics and then using the SCI to track it against a spectrum of what is green IT delivery, amber and red (a very rudimental breakdown of what good and bad looks like!).

I am currently working on a PhD that is focusing on building the framework and methodology that project, product and business managers can use to make decisions during planning and executing technology projects. Before we can create certification, we need to breakdown, the type of technology solutions being built ( UX solutions would be included in the solutions ) and then build on all the underlying questions, data that is required, impact of choices. That would then tease out the key areas of training components.

My view is that we do need the content to underpin the certification. Gap analysis would the first step.

**Reply by VirginieCorraze** [reply]
I think we need to describe the patterns that describe what is expected as Product owner / managers and then build the certification to check that the practices are understood and are applicable in their daily job. 
Indeed it takes time to build the content but it has a large impact potential on the preventive side, shifting left the decisions as early as possible in the process, and involving also the business

for product owners :
- defining the 'Done' criterias of the deliveries, including sustainability criteria like the data end of life/cleanup automation, CO2 measurement, adoption KPIs (if not enough used, this will support potential decommissioning decision)
- engaging the customer and assessing the business impact 
- ensuring the environmental impact assessment is done and sustainability criteria considered for every decision ; document the decisions
- propose alternatives to customers for 'good enough changes' that will use less resources and answer the requirement
- challenging the customer on requirements, to ensure they are fit for purpose and will be used 
- assess impact on the end users' devices 

for project managers, it could be to ensure 
- project KPI include green IT KPIs
- that in every project there is a green initiative in the scoping phase: - either specifically linked to the project scope (specific optimisations or efficient hardware for instance) or to remove/cleanup unused features/data in the product, to compensate the new resources usage 
how does it sound?

**Comment by jawache** [comment]
@holanita is there some way to convert this to a discussion? Used to be a button to flip between the two but can't find it. This is an open ended discussion I would say.

**Comment by holanita** [comment]
@jawache it might be because discussions are not active in this repo. Although I don't know much about discussions tbh so it's best if you guys take care of this.

**Reply by jawache** [reply]
For some reason it appeared, converted to a discussion :)

**Comment by benhamidene** [comment]
> @holanita @jawache following our discussion at the community workgroup yesterday: It would be very valuable to have an e-certification covering the early phases of the solutions design, to take the right decisions and influence from the very beginning: • GREEN BY DESIGN: eco-design practices to adopt at the early stages of the product design (scoping phase, high-level requirements, mock-up)
> 
> • Targeted audience: Product Owner, Product manager, Business analyst
> 
> • How to manage the design and delivery of IT solutions by minimizing environmental impact • The right questions to consider, how to arbitrate • How to determine the arguments to embark the internal stakeholders and customers on a responsible aproach. • How to make trade-offs according to your priorities • Define the key performance indicators and measurement to achieve your goals • Role and responsibly of the Product Owner

I support and welcome this suggestion and propose to help on this topic

**Comment by benhamidene** [comment]
Ok. We agree that the audience should not include UX designers

Regarding content.. I do not have experience in creating content in such a community and I can understand that this would be a lot of work.. I hoped we could make at least make small steps and start evolving sth new.
My concern is mainly that I am still missing a place where I could find as a product owner / manager at least a collection of information / quick start guide to help me integrate eco design in my work & product. GSF would provide a good platform for this .
Is there a way to make such small steps ?

Last point: certification . And this is where I could step out “the good” 😊
I am actually not a big fan of certifications. But I accept thatbthe industry needs a way to “evaluate” skills.
This brings to the core point: what is our expectation to a certification we would provide ?
Just motivating people to start thinking about Green-IT . Or should it be a challenging assessment that really defines a certain level (TBD) of skills & expertise in the area .

I would go for the second one , otherwise we risk “dissuading” the value of the certification.

At least it should a level higher than the practicioner certification …

My 2 cents 😊



From: Asim Hussain ***@***.***>
Date: Monday, 6. March 2023 at 10:07
To: Green-Software-Foundation/learn ***@***.***>
Cc: Anis ***@***.***>, Mention ***@***.***>
Subject: Re: [Green-Software-Foundation/learn] New green software training for product owners, product managers, business analysts (Discussion #165)

Thanks @benhamidene<https://github.com/benhamidene> I think first step for this is to identify the audience for this (rather than the content). Given the answers above and @VirginieCorraze<https://github.com/VirginieCorraze>'s original statement I think her identification of the audience as "Product Managers, Project Managers and Business Analysts" is correct. I was asking about UX Designers since the topic was "Eco Design" and wondering if the intention was to speak to UX Designers but again based on the answers above I don't believe our intention is to speak to UX Designers with this content.

I would suggest we perhaps treat this as a certification rather than creating content.

The nature of the Foundation means it's very hard to create training like we did for learn.greensoftware.org, takes many many months of effort and everything needs to go through multiple levels of consensus review, it's pretty tiring.

@camcash17<https://github.com/camcash17> was exploring the idea of just creating the exam for a topic, we create and administer the exam and point people to existing content for the training. If we realize that some content for some exam questions doesn't exist we can create the missing content, or perhaps even outsource or somehow encourage the community to create the content to fill in the gaps. I much prefer that solution, it's very challenging to get multiple people to agree on content I suspect it's going to be much easier to get multiple people to agree to some exam questions.

So I propose we talk about this project in the context of and exam "Certified Green Software Project/Product Manager" (or some term which accurately defines the role).

If we discover during the creation of the exam that much of the required reading/content doesn't exist we can explore options for filling in the gaps and making sure the content exists. But our goal would be just to administer the exam.

Thoughts?
